src/executables/master_request.c

```diff
@@ -66,6 +66,12 @@
 
 #define SERVER_FORMAT_STRING " %s-Server %s (rel %s, pid %d)\n"
 #define HA_SERVER_FORMAT_STRING " HA-Server %s (rel %s, pid %d)\n"
+/* TODO by Joohok
+ * if there is no need to distinguish between HA-server and nonHA-Server,
```

Comment:
When is the time? What is the requirements? It can be handled by other and should be understandable.

Reply:
I'll revise this comment when the https://github.com/CUBRID/cubrid/pull/4400/files#r1227358257 
discussion is concluded